### DISCUSSION TITLE
Enough redundant stuffs

### DISCUSSION THREAD

**Discussion by Neuron114** [PavelZinchenko/event-horizon-frontier-builds#225] [General]
I genuinely surprised that despite how broken the game currently is, there's always new stuffs being added. I seriously still can't get over with that the game forced you to fight 12 enemies, not to mention the difficulty gaps between level 199 and 200. On top of that, the reward is still not worth it, imagine you have fought literally hundreds of them that use Heavy Plasma Emitter, you only get that one golden pulse cannon. (I still think even in exploration level 195 reward is not enough to cover the current difficulty in lvl 200, but it's just for me, anybody else can give opinion on it). The new red ships are questionably big for vanilla standard we used to know (they got good bonus stat too), plus now we are forced to fight 12 of them past level 199, that is the part when we predict the game is going to be as broken as it is now.

There's also new feature got added such as new big satelite, new Annihilation Ray weapon, m2 ion cannons, new crew ability, new worm tail mechanic (which unintendedly buff some weapon, arguably make tails almost a joke as energy shield), drift speed boost removal, those are okay but remember, the game is still broken, adding those may making the game get even more broken. This is not only me that sees something negative from the new feature has affected the game, we all there
![Screenshot_20240513_001435_572064f74bd5f9fa804b05334aa4f912](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/81498439/f2f03adc-f391-4cef-9362-196ed728dfc8)
![Screenshot_20240513_001500_572064f74bd5f9fa804b05334aa4f912](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/81498439/77c6901d-2de4-4ca7-ad5d-3ce8ca34c454)
![Screenshot_20240513_001523_572064f74bd5f9fa804b05334aa4f912](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/81498439/c45c015c-30e2-4713-8eae-f96eb4369ea8)
![Screenshot_20240513_001622_572064f74bd5f9fa804b05334aa4f912](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/81498439/e935532a-050e-4033-a8f0-1cddef5a9b14)

We EHF community are concerned with our favorite game condition, maybe we should agree that it's best to stop brainstorming with new content, and start to fix what's broken in the game

consider this as our plea for the sake of Event Horizon Frontier

**Comment by IsmollCheeKen** [comment]
It has to be said. While there are good additions, and good suggestions that made it through - like the Star Refund when resetting expands, the new anti-cheat for Arena, and the new features for Duel series, but considering everything else - which is too many rapid additions with little to no balancing, as well as changes on things that never needed to be changed, we would rather have wanted you, the developer, to have actually stopped with the new features when you have made the announcement "no more new features until release". It was going so well.

**Comment by PavelZinchenko** [comment]
What do you think are the most broken thing in the game that should be fixed first?

**Reply by Thirteen0130** [reply]
Перк Doppelganger. Многим он кажется читерным (но меня устраивает, и всё же многие недовольны). Лично я нашёл несколько проблем:
1) При использовании перка на дроновозах и клонах после смерти второго корабля, клоны и дроны сходят с ума, хотя не должны.Решение: сделать чтобы дроны/клоны сходили с ума после 3 смертей материнского корабля(как было изначально у главного корабля).
2) После завершения уровня второй корабль уничтожается с большими, мозолящими глаза спецэффектами, и оставляет после себя трупик.
Решение: сделать, чтобы он просто исчезал с поля, как наёмники.
3) Читерно для титанов. Решение:ограничить перк для титанов (или ещё для линкоров, если игроки захотят). Также сделать, чтобы второй корабль обладал 50% характеристиками изначального.
Примечание: Игроки, если хотите что-то добавить или изменить-говорите. Это всё моё мнение, которое может кардинально отличаться от вашего

**Comment by Thirteen0130** [comment]
> Как вы думаете, какие самые неисправные вещи в игре, которые следует исправить в первую очередь?

Новый Аннигиляционный луч. Он слишком читерный. Серьёзно. Повторяется история с био-плазменной торпедой, когда одна такая могла полностью разрушить базу игрока в пвп. Пвп превратилось в ковбойскую дуэль, кто быстрее выстрелит, тот и победил. Игроки снаряжают Thor Аннигилятором с усилителями дальности, скорости и урона. Играть в пвп нереально раздражает.
Решение: тупо понёрфить Аннигиляционный луч.  Я не знаю как.... Добавить больше потребление энергии или уменьшить урон, добавить больше сопротивление щитам к коррозионному урону... Решайте сами, как будет лучше

**Comment by Bezbasheniy** [comment]
Относительно анигиляциооного луча: предложу перевести его урон в более привычный тепловой/энергетический. А корозионный урон исключить в принципе как механику. Это вернет баланс в относительную норму и плюсом немного разгрузит движок что в конечном итоге приведет к снижению количества багов. Относительно перка клонирования: думаю оптимальным решением будет давать такой перк всем кроме титанов, с 75% характеристиками начальеого корабля либо увеличением требуемых очков капитана (к слову в настоящее время на фрегатах этот перк в принципе не требует таковых) 
По поводу ускорения/скорости: ограничить максимальную скорость и дать возможность большого ускорения для лучшей Управляемости кораблей с большой массой.
По поводу червей повторюсь: оставить первоначальную механику хвоста лиш изменив его колизию по типу колизии дронов (союзные корабли и станцыя проходят сквозь хвост, а враги его таранят).
По поводу арены: требуеться коректно работающий античит. Это значительно уменьшит количество багов вызванных нарушением числовых обозначений относительно оригиналов и уменьшит количество негативных отзывов в целом.
Ну и последнее: после устранения известных багов и изменений некоторых аспектов, провести ребаланс вооружения так как большинство оружия в принципе не применяеться игроками в связи со своей слишком низкой эфективностью, и слишком медленной скоростью полета снаряда который попросту не может угнаться за даже кораблемю среднескоростным кораблем. 
P.s: Это моё субьективное мнение основанное на собственном опыте и общении с другими пользователями проэкта, поэтому прошу не бросаться камнями в случае есле я в чемто был не до конца верен.

**Reply by Thirteen0130** [reply]
Мне нравится новая механика хвоста. В игре всего 2 червя, так что она никакая не читерная или ещё что-то в этом роде. Она забавная, можно повеселиться, и избавляет от множества проблем с коллизиями и застреванием. Она больше дополняет игру с Червями, делая её лучше. Вот раньше достаточно было уничтожить хвост ОДНОЙ бомбой и червь становился бесполезным пушечным мясом. А сейчас он подстать своему могучему имени Jormurgandr и Wormship. Поэтому прошу оставить эту механику.

**Comment by Inedilarion** [comment]
Объективно, плохих нововведений механик я вспомнить не могу. Некоторые не настолько хорошо настроены как хотелось бы (скорость, перки капитанов, выпадение ресурсов учёного), но это скорее вопрос времени и потихоньку поправляется.
Субъективно же меня пугает объём таких изменений, игра очень старая и я в ней очень много лет, но настолько быстрые нововведения, их отмены и изменения мотыляют игру как... Не знаю, как потерявший управление грузовик на серпантине который постоянно сворачивает в пропасть и каждый раз комьюнити приходится присылать сюда посты с требованиями отката/ребаланса изменений чтобы как-то вернуть его на дорогу. Я (и не только я, полагаю) просто уже устал действовать в таком ритме. Как от этого избавиться? Пытаться побольше предсказывать какие изменения произойдут в игре при вводе нововведений и пытаться изначально свести их к минимуму. А то после каждого весёлого нововведения пол комьюнити в дрожь бросает при мысли о том что его не получится откатить/сгладить и что оно может остаться в игре навсегда.

Также добавлю что в игре проблемы с балансом оружия. Из всего огромного количества разнообразнейших интересных пушек играбельными являются процентов 5, наверное. Хотя тут и специфика Фронтира сказывается, из-за наличия огромного количества слабых врагов, дальнобойные лазерные орудия и орудия с АОЕ в принципе эффективней всего остального


Objectively, I can’t remember any bad mechanical innovations. Some are not as well tuned as we would like (speed, captain perks, loss of scientist resources), but this is more a matter of time and is slowly getting better.
Subjectively, the volume of such changes scares me, the game is very old and I’ve been playing it for many years, but such fast innovations, their cancellations and changes are shaking the game like... I don’t know how a truck that has lost control on a serpentine road that constantly turns into an abyss and every since the community has to send posts here demanding a rollback/rebalance of changes in order to somehow return it to the road. I (and not only me, I suppose) are just tired of acting at this rhythm. How to get rid of this? Try to predict more what changes will happen in the game when innovations are introduced and try to initially reduce them to a minimum. And then after every fun innovation, half the community shudders at the thought that it won’t be able to be rolled back/smoothed out and that it could remain in the game forever.

I’ll also add that the game has problems with weapon balance. Of the huge number of various interesting guns, probably 5 percent are playable. Although the specifics of the Frontier are reflected here, due to the presence of a huge number of weak enemies, long-range laser guns and AOE guns are, in principle, more effective than anything else

**Reply by Thirteen0130** [reply]
Прям с языка снял. А ведь точно. Были нововведения, которых мы боялись, что они останутся. Не было ни одного спокойного дня без багов или проблем, которые надо написать для решения. Надо сделать передышку от новшеств, чтоб игроки свыкнулись и прывыкли, а пока просто балансировать игру, исправлять старые проблемы. В "Issues" их там много.

**Reply by Zen9890** [reply]
I think best thing that can be done to adress this is to separate beta into two branches:
Experimental: Pavel's playground to have fun creating new bugs and features, where community feedback is collected operatively regarding new features and nothing is here to stay for 100%, so, as innedi described, people dont have to be in constant fear of something not getting reverted back to previous state
Beta: its purpose is to weed out bugs caused by new features that were accepted in by general public

**Comment by TaipionsSalemClient** [comment]
I, too, wonder why all this stuff gets implemented without stopping for fixing anything...

remove worm tail changes
remove speed changes
remove doppelgänger
fix stuff please before going on

thank you!

[edit:] and auto-remove non-english posts without english translation(!)
I even speak a little bit of russian (chuth chuth), but that's just annoying, I tend to just ignore most of the untranslated stuff and consider it very rude to not even have the decency to run your block of text through google translate before posting, clearly saying "ha, fuck you!" to anyone else

**Reply by Thirteen0130** [reply]
Да почему вам ВСЕМ не нравится новый хвост. Приведите КОНКРЕТНЫЕ ДОВОДЫ, ПРИЧИНЫ, ЧТО ВАС НЕ УСТРАИВАЕТ. Вы ведёте себя как ребёнок, которосу что-то не нравится, а он не может сказать, что именно

Why don't you ALL like the new tail. GIVE SPECIFIC REASONS, REASONS THAT YOU ARE NOT SATISFIED WITH. You're acting like a child who doesn't like something, but he can't tell what it is.

**Comment by IsmollCheeKen** [comment]
I will be going in-depth in this discussion in highlighting problems that the game currently have, I ask for your patience to read it all.



Current issues in the game right now:

1. Features that needs immediate attention:
a. Doppelganger pilot perk
b. Increased speed limit
c. New Afterburner mechanics

2. Bugs:
a. Mercenary pilot bonuses not working on drone bays
b. AI became stupid
c. Provocator pilot perk not working

3. Other stuff:
a. Annihilation Ray is too OP
b. Unfavorable new wormtail mechanics
c. Flawed new Arena system 
d. Old highly voted issues/suggestions yet to be addressed





1.a. Doppelganger Pilot Perk

Aside from the mixed reception on the addition of this pilot perk, the perk itself is causing game breaks in Arena, as confirmed by trusted testers.

![Screenshot_20240513_162005](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/1c4be7ba-6e7d-4ebc-8d8d-2150e9befd9b)

https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/41a5b018-4db2-4317-8f33-f9d7c3c8b060

Errors like this happens even if you don't use doppelgangers on your fleet, the enemy having doppelgangers is enough cause for this bug.

1.b. Increased Speed Limit

Honestly, this addition is very problematic, aside from the negative changes in AI behavior with very high speed ships, it feels as if this change was added on impulse. The 20/20 speed limit was alright and never needed any changes to begin with. Like I always say: "If it's not broken, DO NOT FIX IT"

But if it is so really desired to stay, then here is a proposal on how to balance it:
- Instead of increasing the speed limit, return the limit back to 20/20, and add speed modifiers depending on the game mode;
• PvE Levels = x1 speed (20/20 max)
• Exploration = x2 speed (40/40 max)
• Starbase Battle = x1.5 (30/30 max)
• Duel Series = x1.5 (30/30 max)
• Daily Missions/Citadel Battles/Ship Duel = x1.25 (25/25 max)

1.c. New Afterburner Mechanics

This is another issue as combined with problem 1.b doubles down on the already unstable speed limit. Just like above, this never needed to be changed at all, it worked fine as is before. If it's not broken, don't fix it. My only suggestion for this is to revert it back to its previous mechanics.

2.a Mercenary Pilot Bonuses not Working on Drones

Tested by ChaosFred himself. At some point after build version 1902, drone performance has drastically dropped to levels that would deem it completely unusable, and it still is today.
![Screenshot_2024_0513_164520](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/0f2ce26e-4b2e-49de-8543-4d8cfb132205)
![Screenshot_2024-05-13-00-07-20-552_com ZipasGames Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/90b3e59b-81c4-4479-a1bd-024c19976aab)
![Screenshot_2024-05-13-00-07-06-313_com ZipasGames Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/6e655018-b654-4bde-a792-c6aa8c209e57)

For the sake of beta testing, ignore the fact that he is a game modder. I can assure you that he is a very trusted member of the community.

2.b. AI Became Stupid

An issue about the drastic negative change in AI behavior was made some time ago, and it has only gotten worse.

https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/8a9a74fe-1cb2-441a-b9a1-f3e1e71e95fc

Credits to Magne for providing video footage. The ship is AI-controlled with Autopilot, and as obvious, it is more Artificial than Intelligent.

2.c. Provocator Pilot Perk Not Working Properly

There has been questions raised in the discord about the Provocator pilot perk not working, unfortunately, it failed to gain traction and the issue fell into obscurity. I am mentioning this issue here now to have the perk be taken for a second look. It could be that the perk is not really working or maybe it's because this problem goes hand in hand with problem 2.b.

3.a. Annihilation Ray is Too OP

As the title says, it is still too strong. It brings up the same problem caused by the old Heavy Acid Torpedo - the main reason why HAT was nerfed in the first place. Which is the problem of the Starbase's certain death if ever the Ray lands a hit.

3.b. Unfavorable New Worm Tail Mechanics

Overall, the consensus about this mechanic is mostly negative in the discord server. Why? Because of 3 reasons:

• This change in general weakens Wormships, mainly due to the fact that if a middle segment is destroyed, it takes the other segments behind it with it. In battles, the first tail segment that usually gets destroyed is the one right after the head, you lose that segment - you lose your entire tail.

• This change also indirectly gives a buff to weapons with piercing bullets (like Heavy Plasma Emitter, and Annihilation Ray), and to bullets with AoE effects (Missiles, Holy Cannon, Heavy Ion Cannon, etc.) against Wormships. Since all weapons are gonna be aiming at the Wormships' head, the first tail segment right after head will almost always be destroyed first.

• Lastly, is the intent of the original GitHub issue about this change. The original issue was to simply "make wormship tail not get stuck on the Starbase". Literally just a fix for the tails getting stuck. Instead, we are given an entirely new tail mechanic that was not originally asked for.

3.c. Flawed New Arena System

The new Arena system of not being able to get Tokens unless you battle in ranked matchmaking is quite an unnecessary change. Additionally, an issue regarding the purpose of Arena ratings is pointed out by ChaosFred himself:
![Screenshot_2024_0513_172825](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/164343919/b16abcfe-64f6-476f-858f-119533962adf)
*MM = matchmaking

3.d. Old Highly Voted Issues/Suggestions Yet to Be Addressed

This repository has a backlog of really good, highly favored suggestions for features/balancing that was ignored. Long-standing enhancement suggestions like improvements in Scientist Upgrades, tweaks in the massive difficulty jump the moment you enter Level 200-1, etc. We would like to have these issues be addressed instead of just sitting here collecting dust.
